ocpi-toolkit-2.2.1/src/main/kotlin/com/izivia/ocpi/toolkit/common/HttpUtils.kt

```diff
@@ -243,27 +253,41 @@ fun HttpRequest.parseAuthorizationHeader() = getHeader(Header.AUTHORIZATION)
 /**
  * Throws an exception if the token is invalid. Does nothing otherwise.
  *
- * TODO: is it the good behaviour given:
- * - tokenA: Valid in receiver context, during sender registration (only for sender -> receiver calls)
- * - tokenB: Valid in sender context, during sender registration (only for receiver -> sender calls)
- * - tokenC: Valid when the sender is registered with the receiver (only for sender -> receiver)
- *
  * @throws OcpiClientInvalidParametersException if the token is invalid, otherwise does nothing
  * @throws OcpiClientNotEnoughInformationException if the token is missing
  * @throws HttpException if the authorization header is missing
+ *
  */
-suspend fun PlatformRepository.tokenFilter(httpRequest: HttpRequest) {
+suspend fun PlatformRepository.tokenFilter(
```

Comment:
I would rename that to `checkToken`: we are not building a filter, we really checking the provided token

Reply:
fixed